x-pack/test_serverless/api_integration/test_suites/observability/apm_api_integration/common/apm_api_supertest.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  ApmUsername,
```

Comment:
Do we need to import this? Can't we just have viewer / editor / admin roles defined in serverless test? Same with password, and we could get rid of the eslint disable

Reply:
ATM, we don't need this as the username and password for Serverless are fixed to elastic user and default password. So yes this can be removed